src/components/html/EcApplicationCard.html

```diff
@@ -0,0 +1,22 @@
+<div class="ec-application-card">
+  <div class="ec-application-card__body">
+    <div class="ec-application-card__body__thumb">
+      <img src="https://blog.melhorenvio.com.br/wp-content/uploads/2019/09/Melhor-Envio.png">
+    </div>
+    <div class="ec-application-card__body__content">
+      <div class="ec-application-card__body__content__title">
```

Comment:
```suggestion
      <div class="ec-application-card__title">
```

Não acho legal levar o BEM tão ao pé da letra, acaba ficando ruim de ler e pior ainda de manter, geralmente eu uso só mais um nível de elemento, no máximo dois, depois do prefixo do componente.

Não é este o caso, mas quando o template tá ficando complexo demais e começa a ficar ambíguo mantendo apenas um nível de elemento, eu entendo que deveria criar outro componente e importá-lo...